tasks/human_eval_076.rs

```diff
@@ -5,11 +5,44 @@ HumanEval/76
 /*
 ### VERUS BEGIN
 */
+use vstd::arithmetic::logarithm::log;
+use vstd::arithmetic::power::pow;
 use vstd::prelude::*;
-
 verus! {
 
-// TODO: Put your solution (the specification, implementation, and proof) to the task here
+fn is_simple_power(x: u32, n: u32) -> (ret: bool)
+    requires
+        n > 1,
+    ensures
+        ret <==> x == pow(n as int, log(n as int, x as int) as nat),
+{
+    let maybe_x = n.checked_pow(x.ilog(n));
+    if (maybe_x.is_some() && maybe_x.unwrap() == x) {
```

Comment:
I think you can replace the entire if-else expression with just the conditional, i.e., with `maybe_x.is_some() && maybe_x.unwrap() == x`